packages/api_bo/src/resolvers/mutation.ts

```diff
@@ -642,7 +644,7 @@ export const Mutation = {
         const instructorGroupsIds = instructorGroups.map((group) => group._id);
         const groupsToRemove = instructorGroupsIds.filter(
           (group) => {
-            if (group != undefined) return !groups.includes(group);
+            if (group !== undefined) return !groups.includes(group);
```

Comment:
esto se suele poner `if(!group)`

Reply:
Has dicho que devuelve is group no esta definido, pero si lo está, qué devuelve?

Reply:
no sé si lo que querrías poner es
```typescript
return !groups?.includes(group)
```